relay-server/src/metrics_extraction/event.rs

```diff
@@ -1209,7 +1216,9 @@ mod tests {
 
         assert!(!metrics.is_empty());
         for metric in metrics {
-            if metric.name == "c:spans/count_per_op@none" {
+            if metric.name == "c:spans/count_per_op@none"
+                || metric.name == "d:spans/exclusive_time_light@millisecond"
```

Comment:
Instead of skipping this one, should we assert that the tags are not set on `exclusive_time_light`?

Reply:
Addressed [here](https://github.com/getsentry/relay/pull/2831/commits/d3ab814059f8a20ee66b70a152d643402f4ea42e?diff=split&w=1#diff-d25a118d71db14a3d77408d5cb5a71753d453771343e3fc77d84305036a04e81R1219-R1224)!